src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -125,70 +126,6 @@ public void UpdateVariable(string name, FormulaValue value)
             Recalc(name);
         }
 
-        /// <summary>
-        /// Type check a formula without executing it. 
-        /// </summary>
-        /// <param name="expressionText"></param>
-        /// <param name="parameters"></param>
-        /// <returns></returns>
-        public CheckResult Check(string expressionText, FormulaType parameterType = null)
-        {
-            return CheckInternal(expressionText, parameterType, intellisense: false);
-        }
-
-        private CheckResult CheckInternal(string expressionText, FormulaType parameterType = null, bool intellisense = false)
-        {
-            if (parameterType == null)
-            {
-                parameterType = new RecordType();
-            }
-
-            var formula = new Formula(expressionText);
-
-            formula.EnsureParsed(TexlParser.Flags.None);
-
-            // Ok to continue with binding even if there are parse errors. 
-            // We can still use that for intellisense. 
-
-            var resolver = new RecalcEngineResolver(this, _powerFxConfig, (RecordType)parameterType);
-
-            var binding = TexlBinding.Run(
-                new Glue2DocumentBinderGlue(),
-                formula.ParseTree,
-                resolver,
-                ruleScope: parameterType._type,
```

Comment:
>uleScope: parameterType._type, [](http://example.com/codeflow?start=17&length=30)

'intellisense' parameter is no longer used. It used to be used on this line but was removed. So just remove the parameter altogether.